temperature/router.py

```diff
@@ -0,0 +1,21 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from dependencies import get_db
+from temperature import crud, schemas
+
+router = APIRouter()
+
+
+@router.get("/temperatures/", response_model=list[schemas.Temperature])
+async def get_temperatures(
+    city_id: int | None = None,
```

Comment:
The type hint `int | None` uses the new union operator introduced in Python 3.10. If you need to support older Python versions, consider using `Optional[int]` from the `typing` module.